client/js/_author/components/assessments/question_types/multiple_choice/multiple_choice.jsx

```diff
@@ -29,9 +29,24 @@ class MultipleChoice extends React.Component {
     activeChoice: React.PropTypes.string,
   };
 
+  constructor() {
+    super();
+    this.state = {
+      newChoice: false,
+    };
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (!this.state.newChoice && _.has(nextProps, 'item.question.choices.new')) {
+      this.setState({ newChoice: true });
+    } else if (this.state.newChoice) {
```

Comment:
it needs to be else if because we dont want to setState everyTime component will recieve props